### ISSUE TITLE
Can't run "ruleunit-event-driven-quarkus" tests using VSCode (WSL2) Test Runner for Java extension

### ISSUE THREAD

**Issue by faustool** [apache/incubator-kie-examples#1199]
### Describe the bug

When I try to run the `ruleunit-event-driven-quarkus` project tests in VSCode with a project in WSL using the `Test Runner for Java` extension, it fails with this error at the bottom of the stacktrace:

```
Caused by: java.lang.IllegalArgumentException: container.image.kafka property should be set in pom.xml
 at org.kie.kogito.testcontainers.KogitoGenericContainer.lambda$getImageName$2([KogitoGenericContainer.java:44]())
 at java.base/java.util.Optional.orElseThrow([Optional.java:408]())
 at org.kie.kogito.testcontainers.KogitoGenericContainer.getImageName([KogitoGenericContainer.java:44]())
 at org.kie.kogito.testcontainers.KogitoGenericContainer.([KogitoGenericContainer.java:34]())
 at org.kie.kogito.testcontainers.KogitoKafkaContainer.([KogitoKafkaContainer.java:43]())
 at org.kie.kogito.testcontainers.quarkus.KafkaQuarkusTestResource.([KafkaQuarkusTestResource.java:48]())
```

Everything works fine when I run `mvn clean install`, but not from inside VSCode.

### Expected behavior

I expect all tests to run in the IDE as it runs using Maven.

### Actual behavior

All tests fail with this exception at the bottom:
```
Caused by: java.lang.IllegalArgumentException: container.image.kafka property should be set in pom.xml
 at org.kie.kogito.testcontainers.KogitoGenericContainer.lambda$getImageName$2([KogitoGenericContainer.java:44]())
 at java.base/java.util.Optional.orElseThrow([Optional.java:408]())
 at org.kie.kogito.testcontainers.KogitoGenericContainer.getImageName([KogitoGenericContainer.java:44]())
 at org.kie.kogito.testcontainers.KogitoGenericContainer.([KogitoGenericContainer.java:34]())
 at org.kie.kogito.testcontainers.KogitoKafkaContainer.([KogitoKafkaContainer.java:43]())
 at org.kie.kogito.testcontainers.quarkus.KafkaQuarkusTestResource.([KafkaQuarkusTestResource.java:48]())
```

### How to Reproduce?

Steps:

1. Clone https://github.com/kiegroup/kogito-examples.git inside your WSL 2 installation
2. Open a terminal window
3. Navigate to `kogito-examples/kogito-quarkus-examples/ruleunit-event-driven-quarkus`
4. Run `mvn clean install` in the terminal just to be sure everything runs with Maven
5. Open VSCode using `code .`
6. Wait for the IDE to parse the Java and Maven project
7. Once everything is OK and there are no errors, switch to the `Testing` tab
8. Right click `RuleUnitEventDrivenIT` and click `Run Test`

### Output of `uname -a` or `ver`

Linux USGSOW5G6V5J3 5.10.60.1-microsoft-standard-WSL2 #1 SMP Wed Aug 25 23:20:18 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

### Output of `java -version`

openjdk version "11.0.14" 2022-01-18 OpenJDK Runtime Environment Temurin-11.0.14+9 (build 11.0.14+9) OpenJDK 64-Bit Server VM Temurin-11.0.14+9 (build 11.0.14+9, mixed mode)

### Kogito version or git rev (or at least Quarkus version if you are using Kogito via Quarkus platform BOM)

1.19.0.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.8.5 (3599d3414f046de2324203b78ddcf9b5e4388aa0)

### Additional information

My system:

1. Windows 11
2. WSL 2 (ubuntu)
3. VSCode 1.65.2
4. VS Code extension vscjava.vscode-java-test
5. VS Code extension vscjava.vscode-java-pack
6. VS Code extension redhat.java

I'm sorry if this looks like more like a WSL 2 and VSCode problem, but I can run tests on any other project in this system, except for this, so I don't think WSL or VSCode folks would be able to help here.

**Comment by faustool** [comment]
I looked at `KogitoGenericContainer.java:44` and it throws an exception because this `System.getProperty(propertyName)` returns `null`. In this case, `propertyName` is `container.image.kafka`, and there's nothing in the code loading this property.

So I assume there's some automated task that's executed when running the integration test from `Maven`, but is skipped when running it from the IDE.

**Comment by cristianonicolai** [comment]
Created https://issues.redhat.com/browse/KOGITO-6922 to track this issue.

**Comment by cristianonicolai** [comment]
a fix has been merged, expect it to be available on Kogito 1.20 release.
